go.mod

```diff
@@ -15,14 +15,17 @@ require (
 	golang.org/x/tools v0.1.6-0.20210820212750-d4cc65f0b2ff
 	k8s.io/apimachinery v0.23.5 // for `kubebuilder alpha config-gen`
 	sigs.k8s.io/controller-runtime v0.11.2
-	sigs.k8s.io/controller-tools v0.7.0 // for `kubebuilder alpha config-gen`
-	sigs.k8s.io/kustomize/kyaml v0.10.21 // for `kubebuilder alpha config-gen`
+	sigs.k8s.io/controller-tools v0.8.0 // for `kubebuilder alpha config-gen`
```

Comment:
We are actually here upgrading the `sigs.k8s.io/controller-tools` version used by the config gen so we ought to update the pr title for 

> ✨ (config-gen alpha command): upgrade sigs.k8s.io/controller-tools from v0.7.0 to v0.8.0